test/basic-sdk.js

```diff
@@ -119,6 +119,8 @@ var registry = {
 
   standardSchema: function() {
     invokeCallback(arguments, [null, ['', '', '']]);
+		// Timeout to postpone execution for correct simulation
+    setTimeout(invokeCallback.bind(null, arguments, [null, '0a054261736963123f536368656d612077697468206f6e65206f72206d6f7265206964656e74697469657320616e64206f6e65206e616d6520616e64206465736372697074696f6e1a4c6d657373616765205468696e67207b206f7074696f6e616c20737472696e67206e616d65203d20313b206f7074696f6e616c20737472696e67206465736372697074696f6e203d20323b207d']));
```

Comment:
It is still invoked right away, on line 121. Also callback parameters in timed out callback doesn't seem right. Probably it is a wrong merge.